gemfirexd/shared/src/main/resources/com/pivotal/gemfirexd/internal/loc/toolsmessages.properties

```diff
@@ -545,6 +569,86 @@ FS_EXTRA_HELP=\
 \                 data and will act as an accessor JVM still having the\n\
 \                 capability to route queries to appropriate datastores and\n\
 \                 aggregating the results.\n
+#snappy specific message hides mcast port args, otherwise same as above
+FS_SNAPPY_EXTRA_HELP=\
+\   -sync         If true then wait for complete initialization of the server\n\
+\                 including DataDictionary, persistent tables, network server,\n\
+\                 else return when this node starts waiting for other members\n\
+\                 to join to sync persistent data (default false).\n\
+\   -heap-size    Sets both the initial heap and max heap sizes.\n\
+\                 Size has the same format as the -Xmx/-Xms JVM options.\n\
+\                 -heap-size is equivalent to specifying both -J-Xmx and -J-Xms.\n\
+\   -off-heap-size Size has the same format as the -Xmx/-Xms JVM options.\n\
+\   -locators     The list of locators as comma-separated host:port values used\n\
+\                 to communicate with running locators in the system to\n\
+\                 discover other peers of the distributed system. The list\n\
+\                 must include all locators in use, and must be configured\n\
+\                 consistently for every member of the distributed system.\n\
+\   -start-locator Specified an embedded locator in this server that will be\n\
+\                 started and stopped automatically with this server. The\n\
+\                 locator is specified as <address>[<port>], or just <port>.\n\
+\                 When address is not specified then the one specified in\n\
+\                 "-bind-address" is used, if set, else machine's default\n\
+\                 address is used.\n\
+\   -server-groups A comma separated list of server groups that this JVM is\n\
+\                 a member of. Used for creating tables in particular sets\n\
+\                 of servers or for firing data-aware procedures in particular\n\
+\                 server groups. See the documentation of SERVER GROUPS\n\
+\                 extension to CREATE TABLE and of CALL PROCEDURE\n\
```

Comment:
Remove "CALL PROCEDURE" since that is obsolete mode. Change "SERVER GROUPS" to "SERVER_GROUPS" since that is the option name in SnappyData OPTIONS syntax.